utils/contributors.js

```diff
@@ -59,6 +59,35 @@ const CONTRIBUTERS = [
         `,
         github: "https://github.com/debarati-06",
     },
+    
+    {
+        handle_name: 'Harshith1263',
+        name: "Harshith",
+        image: "/home/harshith/CSOC22-Week0-namecards/public/images/Harshith.jpeg",
```

Comment:
Image path is incorrect. It should be `/images/Harshith.jpeg` instead of what you have mentioned.

Reply:
OKK